great_expectations/cli/v012/suite.py

```diff
@@ -111,7 +111,7 @@ def _suite_edit(
                 batch_kwargs = json.loads(batch_kwargs_json)
                 if datasource:
                     batch_kwargs["datasource"] = datasource
-                _batch = toolkit.load_batch(context, suite, batch_kwargs)
+                toolkit.load_batch(context, suite, batch_kwargs)
```

Comment:
@Kilo59 How about `_ = ...` just to not lose the interpretability that there can be a return value?  Thanks.

Reply:
Wil do.